buildlib/pr/main.yml

```diff
@@ -1,6 +1,6 @@
 variables:
   DOCKER_OPT_VOLUMES: -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools
-  DOCKER_OPT_GPU: --gpus all --ulimit memlock=-1:-1 --device=/dev/infiniband/
+  DOCKER_OPT_GPU: --gpus all --cpus="2.0" --ulimit memlock=-1:-1 --device=/dev/infiniband/
```

Comment:
which CPUs will it use? can it be according to agent it so it will not "collide" with other agents?

Reply:
Currently all agents have CPUAffinity
```
[root@swx-rdmz-ucx-gpu-01 ~]# taskset -pc 23350
pid 23350's current affinity list: 0,1
```

but it doesn't work for docker, because docker launch via API (agent launch /usr/bin/dockerd service ) and cpu affinity does not apply to new docker process

Reply:
and ` --cpus="2.0" ` looks not work

Reply:
Can we use cpuset?
https://docs.docker.com/engine/reference/run/#cpuset-constraint